integration-tests/tests/client/process_blocks.rs

```diff
@@ -104,6 +104,19 @@ pub fn create_nightshade_runtimes(genesis: &Genesis, n: usize) -> Vec<Arc<dyn Ru
         .collect()
 }
 
+pub fn create_test_env(
```

Comment:
suggestion(non-blocking): we might inline/remove this function and call `TestEnv::builder` at the call-site? This function mostly converts named arguments to positional arguments, which hurts readability. Otoh, even if we do want to do that, might be better left to a follow-up.